lumispy/signals/common_luminescence.py

```diff
@@ -21,47 +21,89 @@
 -----------------------------------------------------
 """
 
-from numpy import isnan
+from logging import warning
+from numpy import isnan, array, abs
 from warnings import warn
 
 
 class CommonLumi:
     """**General luminescence signal class (dimensionless)**"""
 
-    def crop_edges(self, crop_px):
+    def crop_edges(self, crop_range, crop_units="pixel"):
         """Crop the amount of pixels from the four edges of the scanning
-        region, from out the edges inwards.
+        region, from out the edges inwards. Cropping can happen uniformily or by specifying the croping range for each axis or each side.
 
         Parameters
         ----------
-        crop_px : int
-            Amount of pixels to be cropped on each side individually.
+        crop_range : int, float, tuple
```

Comment:
renaming breaks the API, we should still accept `crop_px` with a warning that it is deprecated and will be removed in v1.0